Source/WebCore/Modules/mediastream/gstreamer/GStreamerIceAgent.cpp

```diff
@@ -122,6 +122,24 @@ static void webkitGstWebRTCIceAgentSetForceRelay(GstWebRTCICE*, gboolean)
     GST_FIXME("Not implemented yet.");
 }
 
+static void webkitGstWebRTCIceAgentSetRiceStunServer(WebKitGstIceAgent* agent, StringView host, uint16_t port)
```

Comment:
I would use CStringView here.

Reply:
Why? URL::host() returns a StringView. The resolveAddress return value is a String. I see no reason to complicate this with CStringView gymnastics :)